javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javassistmodel/JavassistClassDeclaration.java

```diff
@@ -202,7 +198,7 @@ private String getSuperclassFQN() {
                 }
             }
         } catch (NotFoundException e) {
-            throw new RuntimeException(e);
+            candidates.addAll(solveSuperClass().getAllMethods());
```

Comment:
mmm I am not convinced by this. Why is the "NotFoundException" thrown? I do not think we should ignore it.

Regarding delegating to the super class I would do that later, after the catch clause and do something similiar to what we do in ReflectionClassDeclaration:

```
        if (getSuperClass() != null) {
            ResolvedClassDeclaration superClass = (ResolvedClassDeclaration) getSuperClass().getTypeDeclaration();
            SymbolReference<ResolvedMethodDeclaration> ref = MethodResolutionLogic.solveMethodInType(superClass, name, argumentsTypes, staticOnly, typeSolver);
            if (ref.isSolved()) {
                methods.add(ref.getCorrespondingDeclaration());
            }
        }
        for (ResolvedReferenceType interfaceDeclaration : getInterfaces()) {
            SymbolReference<ResolvedMethodDeclaration> ref = MethodResolutionLogic.solveMethodInType(interfaceDeclaration.getTypeDeclaration(), name, argumentsTypes, staticOnly, typeSolver);
            if (ref.isSolved()) {
                methods.add(ref.getCorrespondingDeclaration());
            }
        }
```